### PR TITLE
Add: global fonts

### PR DESCRIPTION
**Pull request by gxanshu**
If users wish to incorporate a global font into their website, they currently face an issue with the "grapesjs-fonts" plugin. This is because users are required to manually add the font to each part of their site. With the assistance of this PR, users will be able to seamlessly add a global font to the entire page.

here is the [issue](https://github.com/silexlabs/grapesjs-fonts/issues/7)

Also added the i18n translation for the changes.

here is a demo ;)
grapesjs-font-2023-08-17_10 10 35

### CODE REVIEW

File: src/fonts.js [hunk 1]
```diff
@@ -1,318 +1,408 @@
...
+                      }}
+                    >
+                      ${editor.I18n.t('grapesjs-fonts.Remove')}
+                    </button>
+                  </div>
+                </li>
+              `,
+            )}
+          </ol>
+          <h2>${editor.I18n.t('grapesjs-fonts.Global Font')}</h2>
```
Comment (gxanshu) [inline src/fonts.js:267]: i apologize for the mashup, but the main changes goes here

**Comment by gxanshu** [comment]
i apologize for mashup, but i suggest to please review the changes by clicking on commits given above. not from file changes section.

Thanks

**Comment by lexoyo** [comment]
Hello
Thank you for this, I will review it soon !

**Comment by GetLikha** [comment]
I love that, that's awesome

**Comment by lexoyo** [comment]
@gxanshu i will need you to remove the `;` at the end of lines

Please respect the same coding style :)

It's preventing me to do a proper review

**Comment by gxanshu** [comment]
alright

**Comment by lexoyo** [comment]
thank you @gxanshu for the `;`

here are some issues I get testing your code

1. The linter fails now (`npm run lint`)
2. When you load / store the website, you need to handle your data

**Pull request closed without merging**